includes/class-give-db.php

```diff
@@ -195,7 +195,7 @@ public function insert( $data, $type = '' ) {
 		$column_formats = $this->get_columns();
 
 		// Force fields to lower case
-		$data = array_change_key_case( $data );
+		// $data = array_change_key_case( $data );
```

Comment:
@DevinWalker I commented this code to maintain WordPress column naming standard because this will restrict to have `ID` as a column.